pkg/tsdb/cloudwatch/features/features.go

```diff
@@ -0,0 +1,17 @@
+package features
+
+import (
+	"context"
+
+	"github.com/grafana/grafana-plugin-sdk-go/backend"
+)
+
+const (
+	FlagCloudWatchCrossAccountQuerying    = "cloudWatchCrossAccountQuerying"
+	FlagCloudWatchWildCardDimensionValues = "cloudWatchWildCardDimensionValues"
+	FlagCloudWatchBatchQueries            = "cloudWatchBatchQueries"
+)
+
+func IsEnabled(ctx context.Context, feature string) bool {
```

Comment:
this is really neat, does this mean we could have backend feature toggles in our external datasource plugins if we implemented something similar there? 👀 very nifty!